xrpl/core/keypairs/ed25519.py

```diff
@@ -96,4 +96,6 @@ def _public_key_to_str(cls: Type[ED25519], key: ECPublicKey) -> str:
 
     @classmethod
     def _format_key(cls: Type[ED25519], keystr: str) -> str:
+        if len(keystr) < 64:
+            keystr = keystr.zfill(64)
```

Comment:
Is this safe to do security-wise? Or does it lessen the randomness of entropy?

Reply:
I would recommend just throwing an error if they give less entropy than desired. (Or using a securely random number generator to fill the rest, but I'm not sure which random generators are considered secure)

Reply:
I added this because one of my from_entropy test cases was giving back a private key of `EDB6CBAC838DFE7F47EA1BD0DF00EC282FDF45510C92161072CCFB84035390C4D` while xrpl.js gives back `ED0B6CBAC838DFE7F47EA1BD0DF00EC282FDF45510C92161072CCFB84035390C4D` (note the leading zero after `ED` prefix. My thought was that xrpl-py was cutting off the leading zero somewhere so it should be filled. I do not know if it possible to generate keys with less than 63 characters, so I don't think randomness is concern.

Reply:
Ah, that makes sense - good catch :)